installation/production_deb.rst

```diff
@@ -24,24 +24,24 @@ Steps to be executed on all nodes
 ::
 
   # install the server and initialize db
-  sudo apt-get -y install postgresql-9.5-citus
+  sudo apt-get -y install postgresql-9.6-citus
 
   # preload citus extension
-  sudo pg_conftool 9.5 main set shared_preload_libraries citus
+  sudo pg_conftool 9.6 main set shared_preload_libraries citus
 
-This installs centralized configuration in `/etc/postgresql/9.5/main`, and creates a database in `/var/lib/postgresql/9.5/main`.
+This installs centralized configuration in `/etc/postgresql/9.6/main`, and creates a database in `/var/lib/postgresql/9.6/main`.
```

Comment:
Verified that these paths are accurate on Ubuntu 16.04 when I install Postgres 9.6 without Citus. The particular paths differ on RHEL and I did not verify them.